packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx

```diff
@@ -20,15 +20,14 @@ describe.skip('SelectorPlayground', () => {
     }
   }
 
-  it('playground', () => {
-    cy.mount(() => (
-      <div class="bg-gray-100 h-100">
-        <SelectorPlayground
-          eventManager={createEventManager()}
-          getAutIframe={() => createTestAutIframe()}
-        />
-      </div>
-    ))
+  it('populates cy.get by default with a selector of body', () => {
+    const { autIframe } = mountSelectorPlayground()
+
+    cy.spy(autIframe, 'toggleSelectorHighlight')
+    cy.get('[data-cy="playground-method"]').as('method')
+    cy.get('[data-cy="playground-selector"]').as('selector')
+    cy.get('@method').should('contain', 'cy.get')
+    cy.get('@selector').should('have.value', 'body')
```

Comment:
Nit: could remove the aliases and condense this down, the test ids are pretty descriptive as-is

Reply:
Updated here: [`b9d1636` (#19413)](https://github.com/cypress-io/cypress/pull/19413/commits/b9d163621ee8442545e703d273b98f2a8c33c5b3)